modules/osdk-monitoring-prometheus-servicemonitor.adoc

```diff
@@ -15,6 +15,6 @@ generate a ServiceMonitor Custom Resource (CR) based on it.
 
 .Additional resources
 
-- See the
-link:https://github.com/coreos/prometheus-operator/blob/7a25bf6b6bb2347dacb235659b73bc210117acc7/Documentation/design.md#servicemonitor[Prometheus Operator documentation]
-for more about the ServiceMonitor CRD.
+* See the
+link:https://github.com/coreos/prometheus-operator/blob/master/Documentation/design.md#servicemonitor[Prometheus Operator documentation]
```

Comment:
I wouldn't link to master, in case any changes happen and things get moved around that link would not work anymore.